### PR TITLE
Add support for Ubuntu Groovy 20.10

### PR DESCRIPTION
**Pull request by nyanmisaka**
*enable dav1d AV1 for armhf and arm64
*add libvpx encoder
*update libva to 2.9
*update gmmlib to 20.3.2
*update MediaSDK to 20.3

### CODE REVIEW

**Comment by nyanmisaka** [comment]
@joshuaboniface Rebuild jellyfin-ffmpeg should fix this issue https://github.com/jellyfin/jellyfin-ffmpeg/issues/52
```
Version: 4.3.1-2-bullseye
Depends: libx264-160 (>= 2:0.160.3011+gitcde9a93) ...
```

File: build [hunk 1]
```diff
@@ -28,11 +29,11 @@ case ${cli_release} in
     ;;
     'buster')
         release="debian:buster"
-        gcc_version="7"
+        gcc_version="8"
```
Comment (joshuaboniface) [inline build:32]: Nothing should chamge for stable, why?

File: debian/rules [hunk 2]
```diff
@@ -34,7 +34,9 @@ CONFIG := --prefix=${TARGET_DIR} \
 	--enable-libopus \
 	--enable-libtheora \
 	--enable-libvorbis \
+        --enable-libdav1d \
```
Comment (joshuaboniface) [inline debian/rules]: Spacing, but just a nitpick.

**Review by joshuaboniface — APPROVED** [review]
Looks good with a few comments. Will want to thoroughly test the build changes before tagging a release.

File: build (hunk 1 above)
Comment (nyanmisaka) [reply build:32]: It has been updated to gcc 8.3 in Debian buster(stable).
https://packages.debian.org/buster/gcc

It throw me an 404 error when grabbing the gcc 7 package.

Comment (joshuaboniface) [reply build:32]: Thats weird but OK.

File: debian/rules (hunk 2 above)
Comment (nyanmisaka) [reply debian/rules]: Done

**Review by joshuaboniface — APPROVED** [review]

**Pull request merged**